testing/trino-testing-containers/src/main/java/io/trino/testing/containers/TestContainers.java

```diff
@@ -55,4 +61,20 @@ public static String getPathFromClassPathResource(String resourcePath)
                 // and mounted from there.
                 .getResolvedPath();
     }
+
+    public static void exposeFixedPorts(GenericContainer<?> container)
+    {
+        checkState(System.getenv("CONTINUOUS_INTEGRATION") == null, """
+                Exposing fixed ports should not be used in regular test code. This could break parallel test execution.
+                This method is supposed to be invoked from local development helpers only e.g. QueryExecutor.main(),
+                hence it should never run on CI""");
```

Comment:
Should `"""` be on the new line? 
Do we have codestyle checks for this already?

cc @skrzypo987

Reply:
I don't think so. Let's use the old ugly "" + "" for the time being.

Reply:
@skrzypo987 I'll be very upset if we not use fine features which we have for granted.
The other question how such checkstyle will work in case of queries wrapped in `"""` where everyone will tend to place them on the new line? 
I'd left it as is.

Reply:
I am just saying not to use new features, unless maintainers are already using them or it has been discussed and agreed upon. Many developers would like to use `var` but it is discouraged by the maintainers.
I have absolutely no power over that so you should really just ignore my opinion though.

Reply:
to be continued..